README.md

```diff
@@ -82,10 +82,18 @@ Short description.
 
   Collaborated with NASA to develop a shower system that uses 90% less water. The shower has a purification system that makes it possible to reuse the same batch of water. The company now wants to make this technology - initially developed for space - available to consumers.
 
+- [Seabin Project](https://seabinproject.com/) | Australia | -
```

Comment:
I searched and they started as an [Indiegogo campaign](https://www.indiegogo.com/projects/cleaning-the-oceans-one-marina-at-a-time#/) in 2016, so I think we can use this year as funding year.

Reply:
Okay, I will add that. Thanks!